fil_fungible_token/src/token/mod.rs

```diff
@@ -155,7 +155,11 @@ where
         let result = self.transaction(|state, bs| {
             let balance = state.change_balance_by(&bs, owner_id, amount)?;
             let supply = state.change_supply_by(amount)?;
-            Ok(MintReturn { balance, supply: supply.clone() })
+            Ok(MintReturn {
+                balance,
+                supply: supply.clone(),
+                hook_return_data: RawBytes::default(),
```

Comment:
In case there are other hooks (sender?) or similar later let's call this `recipient_data`. This parallels `operator_data` and `token_data`, and I think the fact that it's in response/return is clear from context.

Reply:
ok, renamed it